db/migrate/20220920084842_rename_author_to_user_id.rb

```diff
@@ -0,0 +1,9 @@
+class RenameAuthorToUserId < ActiveRecord::Migration[7.0]
+  def change
+    remove_column :peeps, :author_name
+
+    add_column :peeps, :user_id, :bigint
+
+    add_foreign_key :peeps, :users
+  end
+end
```

Comment:
interesting why you added this file rather than changing the other migrations `CreatePeeps` & `CreateUsers`